src/tools/interop/interop.cpp

```diff
@@ -121,7 +121,9 @@ std::vector<std::string> Urls;
 
 const char* SslKeyLogFileParam = nullptr;
 
+#ifndef QUIC_BUILD_STATIC
 extern "C" void QuicTraceRundown(void) { }
+#endif
```

Comment:
I've been meaning to do this, but I think I'm going to modify the tracing abstraction a bit to make these unnecessary for things that don't need a rundown.

Reply:
https://github.com/microsoft/msquic/pull/1462

Reply:
Once #1462 is in, I can open a new PR with lock-based synchronization for init/deinit and all the CMake code as a less risky incremental checkin.

Reply:
@jeremyong it's in! Feel free to merge main.

Reply:
Just merged it.